relay-server/src/actors/processor.rs

```diff
@@ -2806,8 +2806,8 @@ impl Service for EnvelopeProcessorService {
 
                 tokio::select! {
                    biased;
-                    // TODO(iker): deal with the error when the sender of the channel is dropped.
-                    _ = subscription.changed() => self.global_config = subscription.borrow().clone(),
+
+                    Ok(()) = subscription.changed() => self.global_config = subscription.borrow().clone(),
```

Comment:
I'm still leaving this `Ok(())` to ensure future regressions to the global config service don't result in envelope loss.